mirgecom/euler.py

```diff
@@ -164,43 +164,33 @@ def split_conserved(dim, q):
     """Get the canonical conserved quantities.
 
     Return a :class:`ConservedVars` that is the canonical conserved quantities,
-    mass, energy, and momentum from the agglomerated object array extracted
-    from the state vector *q*. For single component gases, i.e. for those state
-    vectors *q* that do not contain multispecies mixtures, the returned
-    dataclass :attr:`ConservedVars.species_mass` will be set to *None*..
+    mass, energy, momentum, and any scalars, from the agglomerated object array
+    extracted from the state vector *q*.
```

Comment:
mass, energy, momentum, and any scalars, from the agglomerated object array
    extracted from the state vector *q*. For single component gases, i.e. for 
    those state vectors *q* that do not contain multi-species mixtures, the 
    returned dataclass :attr:`ConservedVars.species_mass` will be set to an empty
    array.

Reply:
We should retain the explanation of the handling of non-mixtures. AK asked for this explicitly.

Reply:
Added back.